### ISSUE TITLE
[Feature request] Publish nord theme to kde global theme store

### ISSUE THREAD

**Issue by enderpirate98** [CachyOS/CachyOS-Nord-KDE#1]
I LOVE the nord theme but I hate how it is exclusive to Cachyos/Arch based distros so if I do 1 day have to move away from this FANTASTIC distro I still want the great theme on my distro.

So please publish it because I would LOVE having it on my system!

**Comment by ddh4r4m** [comment]
Will take this into consideration, it's quite tedious due to dependencies, which are easily installed as part of system install of cachyOS itself, so much easier there.

**Comment by qpcheese** [comment]
Yes , please publish on kde store, or provide folder structure for manual install, i never manage to get this fully installed, only partially cus i have no clue where these folders are suppose to go, and KDE dont recognize the tar.gz as a theme package
This is one of the best theme/icon combinations i have ever seen in a distro , i bring it with me on eveny non cachyos install :D

**Comment by enderpirate98** [comment]
@qpcheese  I did write a guide on installing this theme but I do wonder if Plasma 6 has changed some things and if I need to update it or not, anyway here you go: https://github.com/enderpirate98/Cachyos-look-and-feel
